app/Statistics/Repository/IndividualRepository.php

```diff
@@ -97,8 +97,8 @@ private function commonGivenQuery(string $sex, string $type, bool $show_tot, int
         }
 
         $rows = $query
-            ->groupBy(['n_id', 'n_givn'])
-            ->select(['n_givn', DB::raw('COUNT(*) AS count')])
+            ->groupBy(['n_givn'])
+            ->select(['n_givn', DB::raw('COUNT(distinct n_id) AS count')])
```

Comment:
I can't work out what this fixes.  I see the same results with old/new code.

Reply:
On my site it returns 

![screenshot_2019-02-21 statistiken](https://user-images.githubusercontent.com/564393/53159595-6ffa9280-35c6-11e9-9749-28958d0b282d.png)

and after the fix it returns the same result as in 1.7

![screenshot_2019-02-21 statistiken 1](https://user-images.githubusercontent.com/564393/53159632-83a5f900-35c6-11e9-871b-413ff38c9b94.png)

Reply:
You're right.  For my test data (royal family) there was no difference.  But I see it in other data.